### DISCUSSION TITLE
Does anyone know how to access GPU and make the set up in PyCharm community edition?

### DISCUSSION THREAD

**Discussion by jerosarni** [mrdbourke/pytorch-deep-learning#607] [General]
Hi! I was coding all in Pycharm because it was much more comfortable to be. I want to know if there is any chance to setup GPU in Pycharm. 
Thank you!

**Comment by vukotichm** [comment]
I'm following this as I have a similar question. I have experience with both Python and PyCharm but am very new to PyTorch and was wondering why the course isn't presented using an IDE as such. I'm pretty sure that I know the answer but don't want to jump to any  conclusions.

As for your question, have you checked out this:
https://pytorch.org/get-started/locally/

I haven't yet but it looks like a good starting point.
